express/module/StaticModule.cpp

```diff
@@ -349,26 +348,12 @@ std::vector<Express::VARP> StaticModule::onForward(const std::vector<Express::VA
             if (nullptr != inside->mCache) {
                 inputTensor = Executor::getOutput(inside->mCache.get(), inside->mCacheOffset);
             }
-            auto srcDes = TensorUtils::getDescribe(inputTensor);
-            auto des = TensorUtils::getDescribe(mInputTensors[i]);
-            des->dimensionFormat = srcDes->dimensionFormat;
-            mInputTensors[i]->buffer().type = inputTensor->buffer().type;
+            // Resize
             _resizeTensor(mInputTensors[i], inputTensor, mSession.get());
-        }
-        mSession->resize();
-        // Copy
-        for (int i = 0; i < inputs.size(); ++i) {
-            if (nullptr == mInputTensors[i]) {
-                continue;
-            }
-            auto exprInfo    = inputs[i]->expr();
-        auto inside      = exprInfo.first->inside();
-            auto inputTensor = inside->mOutputTensors[exprInfo.second];
-            if (nullptr != inside->mCache) {
-                inputTensor = Executor::getOutput(inside->mCache.get(), inside->mCacheOffset);
-            }
+            // Copy
             mInputTensors[i]->copyFromHostTensor(inputTensor);
```

Comment:
Can't call copyFromHost before the session is resized.

Reply:
Thanks, I revert the last change